app/src/main/java/com/example/vmedvediev/ua21summerdancecamp/UI/bottomnavigation/notes/SwipeToDeleteCallBack.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.vmedvediev.ua21summerdancecamp.UI.bottomnavigation.notes
+
+import android.content.Context
+import android.graphics.*
+import android.graphics.drawable.ColorDrawable
+import android.support.v4.content.ContextCompat
+import android.support.v7.widget.RecyclerView
+import android.support.v7.widget.helper.ItemTouchHelper
+import com.example.vmedvediev.ua21summerdancecamp.R
+
+abstract class SwipeToDeleteCallBack(context: Context) : ItemTouchHelper.SimpleCallback(0, ItemTouchHelper.LEFT) {
```

Comment:
Woooow!

Reply:
https://medium.com/@kitek/recyclerview-swipe-to-delete-easier-than-you-thought-cff67ff5e5f6 :)